Formula/t/twty.rb

```diff
@@ -20,7 +20,7 @@ class Twty < Formula
 
   # see discussions in https://github.com/mattn/twty/issues/28
   # and https://github.com/orakaro/rainbowstream/issues/342
-  deprecate! date: "2024-08-18", because: "uses the old, unsupported Twitter API"
+  disable! date: "2025-01-01", because: "uses the old, unsupported Twitter API"
```

Comment:
Note: `brew info twty` currently shows
```
Deprecated because it uses the old, unsupported Twitter API! It will be disabled on 2025-08-18.
```
But users may not really care.

Reply:
Our documentation is more relevant than that message. 1 year should be the maximum, so (assuming we don't change dates in https://github.com/Homebrew/brew/pull/18551), we may want to edit message so that line uses something like:
* `disabled on or before ...`
* `disabled by ...`

Since we only guarantee at least 6 months for popular formulae (top 1000). Everything else is fair game to be disabled at any time.